opencti-platform/opencti-front/src/components/ItemStatusTemplate.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent } from 'react';
+import Chip from '@mui/material/Chip';
+import makeStyles from '@mui/styles/makeStyles';
+import { ArrowRightAltOutlined } from '@mui/icons-material';
+import { useFormatter } from './i18n';
+import { hexToRGB } from '../utils/Colors';
+
+const useStyles = makeStyles(() => ({
+  chip: {
+    fontSize: 12,
+    lineHeight: '12px',
+    height: 25,
+    marginRight: 7,
+    textTransform: 'uppercase',
+    borderRadius: '0',
+    width: 100,
+  },
+  arrow: {
+    marginRight: 7,
+  },
+  statuses: {
+    display: 'inline-flex',
+    flexWrap: 'wrap',
+  },
+  status: {
+    display: 'inline-flex',
+  },
+}));
+
+interface Status {
+  id: string
+  order: number
+  template: {
+    color: string
+    name: string
+  } | null
```

Comment:
Why can you have a null item in the list ?

Reply:
well seen, fixe

Reply:
Indeed a template can be null everywhere... maybe that's something we should modify

Reply:
A status is not valid without a template If i'm understand the functionnality properly. Right ?